gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/DatasetInfo.java

```diff
@@ -380,65 +446,27 @@ Dataset toPb() {
   }
 
   /**
-   * Returns a builder for the DatasetInfo object given it's user-defined id.
+   * Returns a builder for a {@code DatasetInfo} object given it's identity.
```

Comment:
I do not agree. Across the library I have been careful to refer to `XXXId` classes/objects as "identity" not to make confusion with the `id()` of info classes and the String itself provided by the user to identify the entity (i.e. a job has an user-provided id, it's not called name in BigQuery).